inst/recorder/app.R

```diff
@@ -146,9 +156,11 @@ shinyApp(
       div(class = "shiny-recorder-controls",
         actionButton("snapshot", "Take snapshot"),
         actionButton("exit", "Exit", class = "btn-danger"),
-        textInput("testname", label = "Name of tests", value = "mytests"),
+        textInput("testname", label = "Name of tests",
+          value = ifelse(load_test, "myloadtest", "mytests")),
```

Comment:
Usually I use `ifelse` when vectorization is needed. For cases like this, I would use 

```R
if (load_test) "myloadtest" else "mytests"
```